RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -233,41 +232,36 @@ class VoiceBroadcastPlaybackViewModel: VoiceBroadcastPlaybackViewModelType, Voic
                 
                 self.voiceBroadcastAttachmentCacheManagerLoadResults.append(result)
                 
-                if let audioPlayer = self.audioPlayer {
-                    // Append the chunk to the current playlist
-                    audioPlayer.addContentFromURL(result.url)
-                    
-                    if let time = self.seekToChunkTime {
-                        audioPlayer.seekToTime(time)
-                        self.seekToChunkTime = nil
-                    }
-                    
-                    // Resume the player. Needed after a buffering
-                    if self.state.playbackState == .buffering {
-                        if audioPlayer.isPlaying == false {
-                            MXLog.debug("[VoiceBroadcastPlaybackViewModel] processNextVoiceBroadcastChunk: Resume the player")
-                            self.displayLink.isPaused = false
-                            audioPlayer.play()
-                        } else {
-                            self.state.playbackState = .playing
-                            self.state.playingState.isLive = self.isLivePlayback
-                        }
-                    }
-                } else {
+                // Instanciate audioPlayer if needed.
+                if self.audioPlayer == nil {
                     // Init and start the player on the first chunk
                     let audioPlayer = self.mediaServiceProvider.audioPlayerForIdentifier(result.eventIdentifier)
                     audioPlayer.registerDelegate(self)
                     
                     audioPlayer.loadContentFromURL(result.url, displayName: chunk.attachment.originalFileName)
-                    self.displayLink.isPaused = false
-                    audioPlayer.play()
-                    if let time = self.seekToChunkTime {
-                        audioPlayer.seekToTime(time)
-                        self.seekToChunkTime = nil
-                    }
                     self.audioPlayer = audioPlayer
+                } else {
+                    // Append the chunk to the current playlist
+                    self.audioPlayer?.addContentFromURL(result.url)
                 }
                 
+                // Start or Resume the player. Needed after a buffering
+                if self.state.playbackState == .buffering {
+                    if self.audioPlayer?.isPlaying == false {
+                        MXLog.debug("[VoiceBroadcastPlaybackViewModel] processNextVoiceBroadcastChunk: Start or Resume the player")
+                        self.displayLink.isPaused = false
+                        self.audioPlayer?.play()
+                    } else {
+                        self.state.playbackState = .playing
```

Comment:
This code could be called if the audioplayer is nil.

Reply:
Fixed in the last commit.